### PR TITLE
CFD-158: Full form : Wiki page.

### PR DESCRIPTION
**Pull request by nader77**
#353

### CODE REVIEW

**Comment by nader77** [comment]
Update: Wiki page has all the features specified in the InvisionApp Fixed documents widget to work with all content types, not only discussions.

**Comment by nader77** [comment]
Update: Using the angular Ckeditor for the body field in full form, Workflow video will follow.

**Comment by nader77** [comment]
Body field editor: 
wiki-page

**Comment by nader77** [comment]
This PR handles:
* Adding document widget to Wiki page, task, task-list
* Adding ckeditor to the body field for all content types.

@amitaibu This PR is ready for code review.

**Comment by zero2one** [comment]
We were planned to use the CKeditor module for this. No need to replace it with AngularCK…

WYSIWYG is not part of this sprint…

**Comment by nader77** [comment]
Ok, Just a heads up: Normal CKEDITOR will conflict with Angular-CKEditor, It's hard to implement the both of them together.
Keep this open for reference if you want.

**Comment by nader77** [comment]
I will move the Ckeditor stuff to another branch, Because the rest is for the wiki-page full form

**Comment by nader77** [comment]
Updated Wiki page form:
selection_053

**Comment by nader77** [comment]
@amitaibu PR is ready for code review.

File: capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module [hunk 1]
```diff
@@ -113,6 +113,8 @@ function c4m_document_widget_field_widget_form(&$form, &$form_state, $field, $in
     'data' => array(
       'purl' => url('', $options_context),
       'relatedDocuments' => $default_values[1],
+      // The form id in HTML is with dashes, not underscores.
+      'formId' => str_replace('_', '-', $form_state['build_info']['form_id']),
```
Comment (amitaibu) [inline capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module]: ``drupal_clean_css_identifier`` ?

**Comment by nader77** [comment]
@amitaibu Yes, Fixed.

**Comment by HelenaEksler** [comment]
Closed. Followed up with https://github.com/capacity4dev/capacity4more/pull/378

**Pull request closed without merging**